src/components/dashboard/FaceVerification/screens/VerificationScreen.js

```diff
@@ -76,17 +84,21 @@ const FaceVerification = ({ screenProps }) => {
     exception => {
       const { name } = exception
 
+      // If user has cancelled face verification by own
+      // decision - redirecting back to the into screen
       if (['UserCancelled', 'ForegroundLoosedError'].includes(name)) {
```

Comment:
foregroundloosed should also be considered a failed attempt.
ANY failure after user started the FV process should be considered a failed attempt.
'UserCancelled' is not a failed attemp because user stopped it intentionally

Reply:
you dont have to show an error screen for foregroundloosed, but track it as a failed attempt